proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
+
+Cons
+
+* Duplicate working group participation agreements need to be signed
+* My have duplicate fees depending on the membership level if both MPWG and CN4J/Jakarta are joined
+* Tends to not commit to longer term backwards compatibility 
+
+== CN4J
+Pros
+
+* Allows for parties interested in both MPWG and CN4J/Jakarta to sign one working group participation agreement
+* Addresses IP gap in existing MP development
+* Tends to focus on stability and backwards compatibility 
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
+
+Cons
+
+* Currently has a higher entry cost and overall overhead to run
+* Currently has no zero entry participation model for larger companies to sponsor specification development
+* Introduces a conflict of interest between evolution and support of enterprise customers and their previous investment in Java EE with a shared steering committee
+* Has more process around separate steering, specification and marketing committees
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
+* Has baggage to overcome due to legal requirements to move Java EE to Jakarta 
+* Has still not proven that the process is capable of delivering novel content to end users
```

Comment:
Maybe add something like "in a timely manner", or "with regular consistency".  Something to indicate that we have not produced a repeatable process with Jakarta yet.

Reply:
This is not a valid point. The working groups have predominantly the same people in them so these same people can deliver novel content to users under a combined CNJWG.

Reply:
This  is a valid point because it's still not proven. It's not just people, but there are other differences like different processes and determining how fast Jakarta wants to move vs how fast its existing customer base can move. That latter issue is a huge one. It's going to take time to figure that out.

Reply:
Right, @smillidge.  That was my point as well.  The same people can deliver novel content, but do we have processes in place for the Jakarta EE WG (CN4J WG) to deliver novel content on a regular basis like we are doing with MP?  I think that's the point of this bullet.

Reply:
@jclingan it's not a con because in the CN4J proposal the MP project is free to create its own specification process. If you want to write a credible unbiased pros and cons document to help people decide address the substantive facts in each proposal not add opinion or intentions. If you wish to write a document describing pros and cons that is advocating one of the positions go ahead but don't dress it up as a neutral document.